inlong-dashboard/src/ui/locales/en.json

```diff
@@ -796,5 +797,18 @@
   "components.FieldList.Sink": "SinkField",
   "components.FieldList.CreateSource": "Create SourceField",
   "components.FieldList.CreateSink": "Create SinkField",
-  "components.FieldList.AddField": "Add Field"
+  "components.FieldList.AddField": "Add Field",
+  "pages.Tenant.config.Name": "TenantName",
+  "pages.Tenant.config.Creator": "Creator",
+  "pages.Tenant.config.CreateTime": "CreateTime",
+  "pages.Tenant.config.Description": "Description",
+  "pages.Tenant.New": "Create",
+  "pages.TenantRole.config.Name": "TenantName",
+  "pages.TenantRole.New": "Create",
+  "pages.TenantRole.config.UserName": "UserName",
```

Comment:
Please change all `UserName` to `Username`, because `username` is a single word, thanks.